packages/ui/src/ErrorBoundary.tsx

```diff
@@ -18,7 +17,6 @@ export class ErrorBoundary extends React.Component<ErrorBoundaryProps, State> {
 
   componentDidCatch(error: Error, info: {componentStack: string}) {
     console.warn("[ErrorBoundary] Caught error", error);
-    Sentry.captureException(error);
```

Comment:
Ah, with the onError alert below, we really don't need Sentry in here. Consumers can put the Sentry hooks down there. Nice!